pkg/svcgen/testing/run.go

```diff
@@ -0,0 +1,100 @@
+package testing
+
+import (
+	"go/build"
+	"testing"
+
+	"github.com/bradleyjkemp/cupaloy"
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+	"github.com/spf13/afero"
+)
+
+type Ctx struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Ctx should have comment or be unexported